src/System.Management.Automation/singleshell/Commands/ConsoleCommands.cs

```diff
@@ -248,7 +248,7 @@ protected override void ProcessRecord()
             catch (ArgumentException ae)
             {
                 ThrowError(resolvedPath,
-                    "InvalidCharacetersInPath", ae, ErrorCategory.InvalidArgument);
+                    "InvalidCharactersInPath", ae, ErrorCategory.InvalidArgument);
```

Comment:
JFYI, this will show-up as a FullyQuilifiedErrorId.
I googled and there are no hits with this exact misspelling so it should be ok to change it.